### ISSUE TITLE
Add "powered by matrix" logo

### ISSUE THREAD

**Issue by ShadowJonathan** [vector-im/logos#8]
In the same vein of the ["made for matrix" badge](https://matrix.org/docs/guides/made-for-matrix-badge-guidelines), i think a "Powered by Matrix" badge would also be really helpful to use.

## Reasoning
"Powered by" is different than "Made for", with the latter implying that a particular solution was made to hook into an *existing* matrix ecosystem, while the former implies that it *utilizes* the protocol for it's own usecase, potentially *adding* more to the ecosystem.

To put it another way; "Made for" implies one's drive to make it was matrix specifically, while "Powered by" implies one's drive is external, but matrix helped them get there, using it as a platform.
